README.md

```diff
@@ -117,7 +119,15 @@ $ tree
 #### Upload a file to S3
 
     s5cmd cp object.gz s3://bucket/
-
+    
+- by setting server side encryption (*aws kms*) of the file:
+    
+    `s5cmd cp -sse aws:kms -sse-kms-key-id <your-kms-key-id> object.gz s3://bucket/`
```

Comment:
This format does not seem identical to other examples. Can you please use code block like other examples?

```
s5cmd cp -sse aws:kms -sse-kms-key-id <your-kms-key-id> object.gz s3://bucket/
```

Reply:
once this is done, i'm gonna merge this pr.